app/src/test/java/com/duckduckgo/app/settings/SettingsViewModelTest.kt

```diff
@@ -88,6 +90,8 @@ class SettingsViewModelTest {
 
     private lateinit var testee: SettingsViewModel
 
+    private val autofillFeature = FakeFeatureToggleFactory.create(AutofillFeature::class.java)
```

Comment:
would be good to add a test for `testee.onAutofillSettingsClick()` that checks the correct command is sent when feature enabled and disabled.